packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -377,7 +379,10 @@ export class AtomicResultList implements InitializableComponent {
     }
   }
 
+  private updateBreakpoints = once(() => updateBreakpoints(this.host));
+
   public render() {
+    this.updateBreakpoints();
```

Comment:
During the render is the right time, looking at Stencil's source code, the style is generated & injected right before the render. If we replace it on `componentDidLoad` there's a slight glitch.